samples/bootloader/src/main.c

```diff
@@ -39,6 +39,8 @@ static void validate_and_boot(const struct fw_info *fw_info, uint16_t slot)
 		set_monotonic_version(fw_info->version, slot);
 	}
 
+	nrf_clock_int_disable(NRF_CLOCK, 0xFFFFFFFF);
```

Comment:
Can this be placed into `uninit_used_peripherals` in bl_boot.c instead?

Reply:
Or can we disable the clock altogether (in kconfig)?

Reply:
Disabling in kconfig is not trivial, because we need some clock source for the zephyr kernel, and currently this is the only supported option. Other solutions are long-term.

I'll move it there.

Reply:
The call could not be inside `uninit_used_peripherals` as the nvic had already been disabled at this point. So I added the line just before the interrupts were disabled.